change/@internal-react-components-d422e84c-24e7-4389-ba1f-3bf9f62313e2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Exposed a prop  in SendBox to allow sendbox to render file uploads without implementing onRenderFileUploads",
```

Comment:
This comment is a little misleading. This PR actually renders the file uploads without a render function for the first time.
The extra prop is just a side effect. How about "Add default implmenetation of file upload UI to SendBox"

Reply:
Done